modules/plugin/imagemosaic/src/test/resources/org/geotools/gce/imagemosaic/test-data/heterogeneous_crs_2/indexer.properties

```diff
@@ -0,0 +1,10 @@
+GranuleAcceptors=org.geotools.gce.imagemosaic.acceptors.HeterogeneousCRSAcceptorFactory
+GranuleHandler=org.geotools.gce.imagemosaic.granulehandler.ReprojectingGranuleHandlerFactory
+HeterogeneousCRS=true
+MosaicCRS=EPSG\:4326
+Schema=*the_geom:Polygon,location:String,crs:String,resX:double,resY:double
+ResolutionXAttribute=resX
```

Comment:
Are these resolution attributes needed?

Reply:
Not really in this test case. 
I was using the same dataset from WPS Raster Download where it's used by the GridGeometryProvider.